.konflux/olm-catalog/bundle/Dockerfile

```diff
@@ -1,10 +1,17 @@
 # use non-scratch parent image as dummy for https://issues.redhat.com/browse/KFLUXBUGS-10
-FROM registry.redhat.io/ubi8/ubi-micro:latest
+FROM alpine as base
```

Comment:
Why using `alpine` ? anything coming from docker.io will be ratelimited *and* disallowed by EC.